src/game/game.gd

```diff
@@ -0,0 +1,20 @@
+extends Node2D
+
+onready var mapNode: Node2D = $Map
+onready var characterNode: Node2D = $Characters
+onready var rng = RandomNumberGenerator.new()
+
+var characterTemplate: Resource = ResourceLoader.load("res://game/character/character.tscn")
```

Comment:
I'd recommend removing the `characterTemplate` variable since it's never used in this script and if we ever moved the character scene around we'd have to edit the loading path